src/OpenTelemetry.Api/ActivityContextExtensions.cs

```diff
@@ -29,13 +31,27 @@ namespace OpenTelemetry
     public static class ActivityContextExtensions
     {
         /// <summary>
-        /// Returns a bool indicating if a ActivityContext is valid or not.
+        /// Returns a value indicating whether or not a <see cref="ActivityContext"/> is valid.
         /// </summary>
-        /// <param name="ctx">ActivityContext.</param>
-        /// <returns>whether the context is a valid one or not.</returns>
+        /// <param name="ctx"><see cref="ActivityContext"/>.</param>
+        /// <returns><see langword="true"/> if the context is a valid otherwise <see langword="false"/>.</returns>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        [Obsolete("Use the IsValid method which accepts ActivityContext by reference for better performance.")]
         public static bool IsValid(this ActivityContext ctx)
```

Comment:
Is simply adding `in` to the existing method here backwards compatible?

Reply:
It is source compatible. Meaning at the call site you don't need to specify "in" the compiler will just take care of it for you. But it isn't binary compatible. Changes the signature to a ByRef parameter. So anyone dropping in the SDK at runtime (I'm thinking about auto-instrumentation primarily) I think they would break. Not 100% sure on this.

Reply:
Simply adding `in` would require all the Propagators to override the new `abstract` method with the `in` modifier. So, it would not be backward compatible.